db/keyval/etcd/etcd.conf

```diff
@@ -35,4 +35,7 @@ resync-after-reconnect: false
 
 # Allow to start without connected ETCD database. Plugin will try to connect and if successful, overall resync will
 # be called
-delayed-start: true
\ No newline at end of file
+allow-delayed-start: true
+
+# Number of seconds between ETCD reconnect attempts. Has no use if `delayed start` is turned off
+reconnect-interval: 1
```

Comment:
Why show different value from the default one, here?

Reply:
because it's config file example. Why set default values via the config file?

Reply:
> Why set default values via the config file?

Precisely because of the reason you mentioned: *it's config example* and should mainly serve as documentation of parameters, but also as starting point for users to see what the default value is without going through the code or logs.